episodes/15-carpentries.md

```diff
@@ -86,7 +86,7 @@ Their aligned missions are accomplished by running accessible, inclusive trainin
 Similarities between Data Carpentry, Library Carpentry, and Software Carpentry workshops include:
 
 - a focus on technical skills,
-- a two-day format taught by volunteer instructors, and
+- a two to four day format taught by volunteer instructors, and
```

Comment:
```suggestion
- a two full day or four half day format taught by volunteer instructors, and
```
a bit longer, but is this more accurate? or are some four full days?

Reply:
@ndporter thoughts on this?

Reply:
@brownsarahm this is probably fine. I think there are occasionally 3-day versions that sites stand up, but they're non-standard and your text is more clear on the scope (that it's not 4 full days but always the same volume of content). Thanks!